demo/message_example.ts

```diff
@@ -429,6 +434,57 @@ const initializeLookerEmbed = (runtimeConfig: RuntimeConfig) => {
   }
 }
 
+/**
+ * Do not use any of the following as an example. The following
+ * tests out edge cases for cookieless login,
+ */
+const initializeErrorControls = (runtimeConfig: RuntimeConfig) => {
+  if (runtimeConfig.showDashboard) {
+    const controls = document.querySelector('.error-controls') as HTMLDivElement
+    if (controls) {
+      if (runtimeConfig.useCookieless) {
+        // Test unrecoverable initial connection
+        const error1 = document.getElementById('error-1') as HTMLButtonElement
+        if (error1) {
+          error1.addEventListener('click', () => {
+            // Hide the dashboard
+            renderDashboard({ ...runtimeConfig, showDashboard: false })
+            // Need to wait a bit for current dashboard to unload
+            setTimeout(() => {
+              // sdk=2 is a deliberate mistake. It tells the embedded Looker
+              // application that the embed SDK is being used. In this case
+              // it is not. The Looker application is going to timeout because
+              // it fails to get a handshake. This is considered a coding error
+              // on the part of the embedding application (this) and is unrecoverable.
+              // Not that the Looker displays an explanatory message in the console.
```

Comment:
```suggestion
              // Note that the Looker application displays an explanatory message in the console.
```

I'm guessing you mean `note` not `not` here